Coupled-Cluster/spin_free_CC/polar.py

```diff
@@ -0,0 +1,84 @@
+# A simple Psi 4 script to compute CCSD from a RHF reference
+# Scipy and numpy python modules are required
+#
+# Algorithms were taken directly from Daniel Crawford's programming website:
+# http://sirius.chem.vt.edu/wiki/doku.php?id=crawdad:programming
+# Special thanks to Lori Burns for integral help
+#
+# Created by: Daniel G. A. Smith
+# Date: 7/29/14
+# License: GPL v3.0
+#
+
+import time
+import numpy as np
+from helper_CCENERGY import *
```

Comment:
Completely understandable.  Oh, also, could you provide references to the papers that you pulled equations from? We are still throwing around ideas for the best way to handle references, but my idea would be to include them in the method README. Feel free to weigh in [here](https://github.com/psi4/psi4numpy/issues/9) if you have a different idea!